### PR TITLE
display kml network links sample

### PR DESCRIPTION
**Pull request by tschie**
shows how to display KML that automatically updates and shows network messages

### CODE REVIEW

**Comment by tschie** [comment]
I've asked why we only see two messages for this data. Will update when I find out

**Comment by colinanderson** [comment]
I think you should only see one message with that data.

**Comment by tschie** [comment]
Just re-ran this with the latest build and now I only see one message

File: src/main/java/com/esri/samples/ogc/display_kml_network_links/DisplayKMLNetworkLinksSample.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+      // show an alert when any network link messages are received
+      kmlDataset.addKmlNetworkLinkMessageReceivedListener(kmlNetworkLinkMessageReceivedEvent -> {
+        Alert alert = new Alert(Alert.AlertType.INFORMATION, kmlNetworkLinkMessageReceivedEvent.getMessage());
+        alert.setHeaderText("KML Network Link Message");
+        alert.initOwner(sceneView.getScene().getWindow());
+        alert.show();
+      });
+
+      // add the KML layer as an operational layer and zoom to it after it is loaded
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/display_kml_network_links/DisplayKMLNetworkLinksSample.java]: Would say, // add the KML layer as an operational layer and check if loaded correctly
This part has nothing to do with setting zoom

File: src/main/java/com/esri/samples/ogc/display_kml_network_links/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<h2>How to use the sample</h2>
+
+<p>The data shown should refresh automatically every few seconds. A network message will be displayed when the dataset is loaded.</p>
+
+<h2>How it works</h2>
+
+<p>To show KML with network links and display network messages:</p>
+
+<ol>
+    <li>Create a <code>KmlNetworkDataset</code> from a KML source which has network links.</li>
```
Comment (john0005) [inline src/main/java/com/esri/samples/ogc/display_kml_network_links/README.md]: This should just be KmlDataset.

**Review by john0005 — CHANGES_REQUESTED** [review]
A minor change.

**Pull request merged**